modules/mono/glue/GodotSharp/GodotSharp/Core/Color.cs

```diff
@@ -627,7 +627,9 @@ private static Color FromHTML(string rgba)
             }
             else
             {
-                throw new ArgumentOutOfRangeException("Invalid color code. Length is " + rgba.Length + " but a length of 6 or 8 is expected: " + rgba);
+                string text = "Invalid color code. Length is " + rgba.Length;
```

Comment:
Should we use interpolated strings instead?
```suggestion
                string text =  $"Invalid color code. Length is {rgba.Length}, but a length of 6 or 8 is expected: {rgba}";
```